readthedocs/doc_builder/templates/doc_builder/data.js.tmpl

```diff
@@ -1,10 +1,10 @@
 var READTHEDOCS_DATA = {{ data_json|safe }}
```

Comment:
Do we not need it here, too?

Reply:
Let me check. There's a [new templatetag](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#json-script) in Django 2 for JSON data but we can't use it yet.

Reply:
There is one in 1.11 https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#escapejs

Reply:
oh, this is a json object I guess?

Reply:
So the above is not quite right. This is a bit complex so I'm going to spend slightly more time here and try to write a test or two.

Reply:
> oh, this is a json object I guess?

Correct. Escapejs doesn't work for this kind of object.